administration/src/components/applications/ApplicationUserOverviewController.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import { NonIdealState, Spinner } from '@blueprintjs/core'
```

Comment:
As you stated in Sarah's PR, we should not use blueprint here.
However, as I commented, we can also refactor this later.

Reply:
adjusted stuff to be responsive